template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/utils/BlueprintUtils.java

```diff
@@ -33,8 +33,8 @@ public class BlueprintUtils {
     // regex for alphanumeric characters and underscores
     private final Pattern validHostGroupNamePattern = Pattern.compile("^\\w+$");
 
-    public String readDefaultBlueprintFromFile(String version, String[] split) throws IOException {
-        return FileReaderUtils.readFileFromClasspath(String.format("defaults/blueprints/%s/%s.bp", version, split.length == 2
+    public String readDefaultBlueprintFromFile(String version, String[] split, String path) throws IOException {
+        return FileReaderUtils.readFileFromClasspath(String.format("defaults/" + path, version, split.length == 2
```

Comment:
if you use the String.format add `%s` or just remove the String.forrmat

Reply:
here `version` and `split.trim` are used as arguments for `String.format`, `path` is part of the 1st argument of `String.format(String format, Object... args)`

Reply:
BTW, no `%s` in the pattern, I don't know how it fills in the string

Reply:
`path` will have it